src/lua/pmanager.lua

```diff
@@ -386,10 +386,27 @@ local function selectPackage()
 		return
 	end
 
-	local docpath = packageInfo(comboboxPackages.currentText).path
-	docpath = docpath.."doc"..s.."index.html"
+	local docpath = packageInfo(comboboxPackages.currentText).path..s.."doc"
+	docButton.enabled = true
+	if not Directory(docpath):exists() then
+		docButton.enabled = false
+		local msg = string.format(
+			"Warning: The documentation of package '%s' was not found.\nPlease execute 'TerraME -package %s -doc' to generate the documentation.",
```

Comment:
Just say that the documentation was not found, removing the part `"Please execute..."` because most of the users do not know that. Only a user that knows how it works might found this situation. In this case, she already knows what to do.